addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2619,6 +2619,8 @@ msgstr ""
 #: xbmc/PlayListPlayer.cpp
 #: xbmc/profiles/dialogs/GUIDialogLockSettings.cpp
 #: xbmc/pvr/dialogs/GUIDialogPVRGuideSearch.cpp
+#. Label for Yes button in a dialog when a user wants to delete a timer that was scheduled by a repeating timer
```

Comment:
I'm not sure if really required or not by our strings.po parser (or the one on transifex), but we have the contextual notes usually above the usage comments. So just to be sure you should probably move that a couple lines up. And as the label now is no longer specific to the timer dialog, it should be changed to a more generic description, like
`#. Label used for buttons and select options as a reference to match all items or ocurrences`

Thanks